.github/workflows/spectral-publish.yml

```
@@ -0,0 +1,40 @@
+name: Publish IPA Validation Framework Package
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - 'tools/spectral/ipa/**'
```

```suggestion
  push:
    branches:
      - main
      - CLOUDP-CLOUDP-329998
    paths:
      - 'tools/spectral/ipa/**'
```
This should make it runnable on this branch, we can do this change temporarily to try it out (Matt tested on another branch)